docs/docker/README.md

```diff
@@ -121,6 +121,7 @@ docker service create --name="minio-service" \
   --env="MINIO_SECRET_KEY_FILE=my_secret_key" \
   minio/minio server /data
 ```
+`MINIO_ACCESS_KEY_FILE` and `MINIO_SECRET_KEY_FILE` also support custom absolute paths, in case Docker secrets are mounted to custom locations or other tools are used to mount secrets into the container (e.g. HashiCorp Vault).
```

Comment:
Can we also specify an example here? Maybe `MINIO_ACCESS_KEY_FILE=/vault/secrets/minio-access-key` to read the access key from vault.

Reply:
@unlimitedbits can you add this example pls?

Reply:
done